src/styles/pages/services/services.module.scss

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Jost:wght@500&display=swap');
+
+.mainContainer {
+    font-family: 'Jost', sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+
+.servicesContainer {
+    background-color: #FFFBF0;
+    color: #5B1110;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 4rem;
+    row-gap: 1rem;
+
+    .heading {
+        padding: 1rem;
+        font-size: 1.6rem;
+        word-wrap: break-word;
+    }
+}
+
+.boxesContainer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 36%;
+    column-gap: 2rem; //do not change gap size 
+
+    .serviceBox {
+        background-color: #E7D3B8;
+        border-radius: 0.75rem;
+        padding: 2rem;
+        text-align: center;
+        flex-basis: 100%;
+
+        >h3 {
+            size: 200%;
+            color: #5B1110;
+            flex-basis: 100%;
+
+        }
+
+        >p {
+            color: black;
+            padding-top: 1rem;
+        }
+
+        >ul {
+            color: black;
+            padding-top: 1rem;
+            columns: 2;
+        }
+    }
+}
+
+// Slider code 
+
+.slideContainer {
+    background-color: #FFFBF0;
+    display: flex;
+    flex-direction: column;
+    color: #D79C1A;
+}
+
+.tabContainer {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    background-color: #FFFBF0;
+
+}
+
+.tab {
+    background-color: #E7D3B8;
+    border: none;
+    row-gap: 1rem;
+    color: #5B1110;
+    padding: 1rem 0rem 1rem 2rem; //top right bottom left 
+    border-radius: 1rem 1rem 0 0;
+    width: 20%;
+    word-wrap: break-word;
+    text-align: left;
+    height: 4rem;
+}
+
+.tab:hover {
+    color: #D79C1A;
+    background-color: #5B1110;
+}
+
+.activeTab {
+    color: #D79C1A;
+    background-color: #5B1110;
+}
+
+.inactiveSlide {
+    display: none;
+}
+
+.activeSlide {
+    display: flex;
+    flex-direction: row; //for some reason using row shows it as column?
+    background-color: #5B1110;
+    padding: 3rem;
+    word-wrap: break-word;
+    width: 100%;
+    font-size: 1.2rem;
+    justify-content: space-around;
+    height: 30rem;
+    align-items: center;
+    position: absolute;
+
+    .arrContainer {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+        width: 5rem;
+        //position: absolute;
+    }
+
+    .arr {
+        height: 1rem;
+        width: 1rem;
+        height: 3rem;
+        width: 3rem;
+        border-top: 0.5rem solid #E7D3B8;
+        border-right: 0.5rem solid #E7D3B8;
+        transform: rotate(45deg);
+        justify-content: center;
+        cursor: pointer;
+    }
+
+
+    .imageContainer {
+        position: relative; //absolute 
+    }
+
+    .image {
+        border-radius: 0.75rem;
+        transform: scale(0.8);
+    }
+
+    .slideText {
+        position: relative;
```

Comment:
here, you can add a width property so that all slides align nicely when you switch through the tabs 👍🏼